RPIPs/rpip-balance-sub-guard.md

```diff
@@ -0,0 +1,50 @@
+---
+rpip: #<to be assigned>
+title: Balance Submission Guardrail
+description: <Description is one full (short) sentence>
+author: knoshua (@knoshua)
+discussions-to: <URL>
+status: Draft
+type: Protocol
+category (*only required for Protocol ): Core
+created: 2024-05-27
+
+---
+
+## Abstract
+This proposal limits how much rETH rate can change with one update. The limit is a protocol parameter controlled by the pDAO. The pDAO can't set it lower than a minimum value. The parameter can't be changed by the Security Council.
+This proposal also limits how frequently updates can happen, using an existing protocol parameter that the pDAO controls. 
+
+## Motivation
+Currently, the oDAO can change rETH rate arbitrarily within one block. This exposes rETH holders: In the worst case, a compromised oDAO can make rETH worthless for any holder and extract any ETH available. During normal operation, the rETH rate doesn't need to change a lot. This proposal aims to reduce oDAO trust by introducing a sensible limit that doesn't interfere with legitimate use of balance updates and removes unnecessary risk from illegitimate use.
+
+## Specification
+
+### Maximum rETH Delta Protocol Parameter
+- There SHALL be a Maximum rETH Delta protocol parameter that the pDAO can vote to change.
+- Maximum rETH Delta SHALL initially be set to 2 percent.
+- The pDAO SHALL NOT be able to set Maximum rETH Delta lower than 1 percent.
+- The Security Council SHALL NOT be able to change Maximum rETH Delta.
+
+### Network Balance Updates
+- Network Balance updates SHALL NOT be able to be sumbitted more frequently than specified by the existing protocol parameter Network Submit Balances Frequency.
+- Network Balance updates SHALL NOT be able to change the rETH exchange rate by more than Maximum rETH Delta.
```

Comment:
I'm going with:
> The oDAO SHALL NOT be able to change the rETH exchange rate by more than Maximum rETH Delta.

I think the "percent" is unnecessary here because Maximum rETH Delta is already defined as a percent value above.